app/controllers/posts_controller.rb

```diff
@@ -236,6 +236,9 @@ def update
       opts[:skip_validations] = true
     end
 
+    # Bypass bump for whisper edits
+    opts[:bypass_bump] = true if post.whisper?
```

Comment:
Rather than passing it through from the controller, I think it would make more sense if the `PostRevisor` had this logic built into its `bump_topic` method. `PostRevistor` is used outside of controllers and we want this behaviour to run everywhere.

Reply:
That was also where I had put it in the first place but since `bypass_bump?` already makes a decision based on the `@opts[:bypass_bump]` it receives, I thought it would be a better way in just setting this variable to true instead of adding more logic to it.